baldrick/github/github_api.py

```diff
@@ -70,20 +70,30 @@ def _headers(self):
     def _url_contents(self):
         return f'{HOST}/repos/{self.repo}/contents/'
 
-    def get_file_contents(self, path_to_file, branch=None):
-        if not branch:
-            branch = 'master'
+    def get_file_contents(self, path_to_file, branch='master'):
+        global file_cache
```

Comment:
Why the global? You don't need it since you are modifying it inplace

Reply:
Explicit is better than implicit? :grinning:

Reply:
Well I still prefer to avoid this if possible and would prefer to use e.g. uppercase notation, but I'll leave it up to you